### ISSUE TITLE
Juvenes restaurants at Linnanmaa campus

### ISSUE THREAD

**Issue by huttunensami** [jtiala/ruoka-client#11]
Amica has left Linnanmaa campus and Juvenes is about to open new restaurants (http://www.juvenes.fi/fi-fi/ravintolatjakahvilat/opiskelijaravintolat/oulunkampus.aspx).

Would it be possible to include their menus in Ruoka.net service? It seems that the menus could be read, for example, from http://www.juvenes.fi/DesktopModules/Talents.LunchMenu/LunchMenuServices.asmx.

**Comment by jtiala** [comment]
Yes! I've already started working on a new backend that supports Juvenes, but I'm too busy with real work atm. Hopefully I'll have something ready by the end of the month, but don't hold your breath.
